packages/lit-helpers/package.json

```diff
@@ -33,6 +33,8 @@
   },
   "devDependencies": {
     "@open-wc/testing": "^2.4.2",
+    "htm": "^2.2.1",
+    "preact": "^8.5.1",
```

Comment:
why is there a devDependency on htm and preact?

what are we testing for? for what reason?

Reply:
You are right. I just retrieve the unit test created in carehtml but I'm going to remove those not related with lit-element/lit-html removing those dependencies.